synfig-studio/src/gui/docks/dockmanager.cpp

```diff
@@ -421,7 +419,6 @@ Gtk::Widget* DockManager::read_widget(std::string &x)
 	// paned
 	if (hor || vert)
 	{
-		// skip "[hor|" or "[vert|"
```

Comment:
Not saying that this kind of comment has a right to live, but it's not commented code either.